cirq/google/gate_sets.py

```diff
@@ -83,5 +83,9 @@
         MEASUREMENT_DESERIALIZER,
     ],
 )
-
 document(XMON, """Gate set for XMON devices.""")
+
+NAMED_GATESETS = {
```

Comment:
Why this name? Are other gates sets not named?

Reply:
named as in string names. The others are just python objects

Reply:
Nit: maybe we want `STRING_NAMED_GATESETS'